backend/src/main/java/com/woowacourse/gongseek/article/application/ArticleService.java

```diff
@@ -56,10 +53,10 @@ public ArticleIdResponse save(AppMember appMember, ArticleRequest articleRequest
         Member member = getAuthor(appMember, articleRequest);
 
         Tags tags = new Tags(getTags(articleRequest.getTag()));
```

Comment:
getTages 메서드에서 수행하고 있는 변환 로직을 Tags 일급컬렉션 내부에서 수행하도록 변경할 수 있겠네요! 그러면 해다 Service 클래스에서 getTages는 삭제해도 되겠네요


```suggestion
        Tags tags = Tags.from(articleRequest.getTag());
```

```java
public static Tags from(List<String> tag) {
    return new Tags(tag.stream()
            .map(Tag::from)
            .collect(Collectors.toList()));
}
```

Reply:
저도 getTags로직이 일급컬렉션에 있는 것이 좋다고 생각해요!

Reply:
수정하였습니다!